pages/common/lighthouse.md

```diff
@@ -0,0 +1,36 @@
+# lighthouse
+
+> Analyzes web applications and web pages, collecting modern performance metrics and insights on developer best practices.
+> More information: <https://github.com/GoogleChrome/lighthouse>.
+
+- Generate an HTML report for a specific website and save it to a file:
+
+`lighthouse {{https://example.com}}`
```

Comment:
Can you configure the file it saves to?

Reply:
That's the 3rd example.

Reply:
Then it might be good to say the name of the file that is generated, or at least pattern. What it is?

Reply:
This is the final command output with the filename:
```
status Generating results... +1ms
Printer html output written to /tmp/www.google.com_2021-06-10_11-39-37.report.html +47ms
ChromeLauncher Killing Chrome instance 11817 +1ms
```

Reply:
How about this then?
```suggestion
- Generate an HTML report for a specific website and save it to a temporary file:

`lighthouse {{https://example.com}}`
```

Reply:
Ah, okay, yes, if that's more accurate it's a lot better 🙂